packages/core/src/v2/filter/filter-function.ts

```diff
@@ -0,0 +1,95 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
```

Comment:
for some reason the git move did not work for this file.

Reply:
There are some files in the v2 which are marked as new files and not as moved. But I get all content there is just a move from the old location. So perhaps you can check this, because this makes the review easier.